.rhcicd/grafana/grafana-dashboard-notifications-general.configmap.yaml

```diff
@@ -4518,6 +4519,202 @@ data:
             "align": false
           }
         },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "PDD8BE47D10408F45"
```

Comment:
`"datasource": { "uid": ...}` should be `"uid": "$datasource"` like others.
It will allow to change targeted env easily.
(same comment for 9 others occurences of this section :) )

Reply:
Good catch! Thanks!